packages/account/src/Sections/Security/ConnectedApps/__tests__/connected-apps-info.spec.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import ConnectedAppsInfo from '../connected-apps-info';
+
+describe('<ConnectedAppsInfo />', () => {
+    beforeEach(() => {
+        render(<ConnectedAppsInfo />);
+    });
+
+    it('should have h4 element with text "What are connected apps"', () => {
+        const heading = screen.getByRole('heading', { name: 'What are connected apps?' });
+        expect(heading).toBeInTheDocument();
+    });
+    it('should have an ordered list', () => {
+        const orderedlist = screen.getByRole('list');
+        expect(orderedlist).toBeInTheDocument();
+    });
+    it('should have three list items', () => {
+        const listitems = screen.getAllByRole('listitem');
+        expect(listitems).toHaveLength(3);
+    });
+
+    it('displays connected apps information', () => {
```

Comment:
```suggestion
    });
    it('displays connected apps information', () => {
```

Reply:
up to you buddy :)
just noticed, that somewhere there is space between tests, and somewhere no. you can skip this :)

Reply:
aint gonna leave that. spacing > coding ⚡ 🔥